config/default.yaml

```diff
@@ -232,5 +232,7 @@ ecosystem:
     badges: https://img.shields.io/badge/build-{{status}}-{{color}}.svg
     # Default registry to pull build containers from. Uses Docker Hub if nothing/empty string is provided
     dockerRegistry: ""
+    # Hostname for metrics service, or pushgateway
+    metric_host: http://pushgateway.example.org:9091
```

Comment:
Should this be camelCase instead of snake_case?

Reply:
Good point, I'll fix that.